tests/unit/BanManager.d

```diff
@@ -41,7 +43,7 @@ private void main ()
     assert(banman.isBanned(IP));
     banman.dump();
 
-    auto new_banman = new BanManager(conf, new Clock(null, null), path);
+    auto new_banman = new BanManager(conf, new Clock(null, null), buildPath(path, "banned.dat"));
```

Comment:
Isn't it right to use the different file?

Reply:
No, the test explicitly tests loading.

Reply:
we use the same file: path was just a directory before and the filename was always banned.dat

Reply:
Got it, ty.